src/lib/components/tablev2/Tablev2.component.js

```diff
@@ -97,12 +116,19 @@ function Table({
     useRowSelect,
   );
 
+  useEffect(() => {
+    if (globalFilter != undefined) {
```

Comment:
```suggestion
    if (globalFilter !== undefined) {
```

Reply:
it's actually to avoid to write `globalFilter !== undefined && globalFilter !== null &&  globalFilter !== ''`

Reply:
Then just `!globalFilter` would do the trick